.github/workflows/test_sdxl.yml

```diff
@@ -21,36 +23,70 @@ jobs:
       - name: "Checkout Code"
         uses: actions/checkout@v4
         with:
-          ref: ean-sd-fp16
+          ref: bump-punet-tom
+      
+      - name: Set up CMake
+        uses: jwlawson/actions-setup-cmake@v1
+
+      - name: Set up Ninja
+        uses: ashutoshvarma/setup-ninja@v1
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
 
       - name: "Checkout iree-turbine"
         uses: actions/checkout@v4
         with:
           repository: iree-org/iree-turbine
-          # TODO: Let the ref be passed as a parameter to run integration tests.
           path: iree-turbine
+      
+      - name: "Checkout iree"
+        uses: actions/checkout@v4
+        with:
+          repository: iree-org/iree
+          path: iree
 
-      - name: Sync source deps
-        # build IREE from source with -DIREE_BUILD_TRACY=ON if getting tracy profile
+      - name: Python deps
         run: |
+          python3.11 -m venv sdxl_venv
+          source sdxl_venv/bin/activate
           python -m pip install --upgrade pip
-          # Note: We install in three steps in order to satisfy requirements
-          # from non default locations first. Installing the PyTorch CPU
-          # wheels saves multiple minutes and a lot of bandwidth on runner setup.
-          pip install --no-compile --index-url https://download.pytorch.org/whl/cpu \
-            -r ${{ github.workspace }}/iree-turbine//pytorch-cpu-requirements.txt
-          pip install --no-compile --upgrade -r ${{ github.workspace }}/iree-turbine/requirements.txt
-          pip install --no-compile -e ${{ github.workspace }}/iree-turbine/[testing,torch-cpu-nightly]
-          pip install --no-compile --upgrade -r models/requirements.txt
-          pip install --no-compile -e models
+          pip install --no-compile -r ${{ github.workspace }}/iree-turbine/pytorch-cpu-requirements.txt
+          pip install --pre --upgrade -r ${{ github.workspace }}/iree-turbine/requirements.txt
```

Comment:
Do we need to install `iree-turbine` explicitly since it's listed as part of `models/requirements.txt`?

Reply:
Yeah not needed